src/main/java/edu/ucsb/cs156/gauchoride/entities/RideRequest.java

```diff
@@ -0,0 +1,30 @@
+package edu.ucsb.cs156.gauchoride.entities;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.AccessLevel;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "riderequests")
+public class RideRequest {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+  private Long userid;
+  private String day;
```

Comment:
Please add a comment indicating whether this is MTWRFSU, Mon/Tue/Wed/Thu/Fri/Sat/Sun or fully spelled out days of week (and specify the capitalization if you choose that.)